project-loader/src/main/kotlin/Loader.kt

```diff
@@ -10,7 +10,8 @@ import java.io.File
 
 data class Plugin(
         val id: String,
-        val path: String
+        val path: String,
+        val isPreInstalled: Boolean?=null
```

Comment:
Why is this a nullable? Wouldn't it make more sense to make it a normal bool and make it false by default? This would allows us to get rid of the null checks below.

Reply:
I am not soo much familiar with Kotlin. I though we could make this property optional, so users will not need to update their checkmodels configuration after switching to a newer version.